### PR TITLE
Fake news detection web application

### PR DESCRIPTION
**Pull request by asiskumarroy**
# Description

A web application built using flask to check the reality of a news article based on US 2016 presidential election

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [x] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [x] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [x] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by asiskumarroy** [comment]
As requested all the requirements are fulfilled along with the removal of errors.Please give it a go.I hope you like it

**Comment by asiskumarroy** [comment]
Hello my request hasnt been merged yet.May i know if any issue is pertaining

**Review by AdityaJ7 — CHANGES_REQUESTED** [review]
Hi @asiskumarroy thanks for being patient 

Your folder must contain only these four things:

1. Your scripts
2. Readme file[structure it as per the readme template ]
3. requirements.txt file [please change yours by referring to the existing scripts in the repo]
4. screenshot of your script/program in action[can have multiple screenshots]

Remove the train and test csv files and provide a  link in the readme from where this dataset can be obtained 

Comment your code in required areas, if not done already

If possible remove the pipeline file as its around 15 MB in space and we want to keep the repo size as small as possible, mention steps in the readme to generate this file.

Thanks

**Comment by tusharnankani** [comment]
Are you working on it? @asiskumarroy

**Comment by asiskumarroy** [comment]
Okay will do the following

**Comment by asiskumarroy** [comment]
As per your request the changes has been made.Please have a look

**Comment by asiskumarroy** [comment]
The changes has been made.Please have a look.

**Comment by asiskumarroy** [comment]
Removing the pipeline will lead to gross errors and it will be quite difficult to mention it in the readme and the reader might not be able to understand how to generate it.I suggest its better be there

**Comment by asiskumarroy** [comment]
Please do let me know if I can be of any help further.

File: Scripts/Miscellaneous/Fake_news_web/Requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: python 3.7
```
Comment (chavarera) [inline Scripts/Miscellaneous/Fake_news_web/Requirements.txt:1]: add used module list in requirements.txt with help of pip freeze

```
pip freeze list>requirements.txt
```

**Review by chavarera — CHANGES_REQUESTED** [review]

**Comment by asiskumarroy** [comment]
Ok will do

File: Scripts/Miscellaneous/Fake_news_web/Requirements.txt (hunk 1 above)
Comment (asiskumarroy) [reply Scripts/Miscellaneous/Fake_news_web/Requirements.txt:1]: ok will do

Comment (asiskumarroy) [reply Scripts/Miscellaneous/Fake_news_web/Requirements.txt]: As per your request module list has been added in requirements.txt using the command
pip freeze list>requirements.txt

**Comment by asiskumarroy** [comment]
The requirements.txt has been updated as per the request.
It has been updated using the command pip freeze list>requirements.txt which adds all the module list.
Do let me know about further changes
Thanks in advance.

**Review by Apex-code — APPROVED** [review]

**Review by AdityaJ7 — APPROVED** [review]

**Comment by AdityaJ7** [comment]
@allcontributors please add @asiskumarroy for code

**Pull request merged**